### PR TITLE
add requirements for being listed on Cilium training

### PR DESCRIPTION
**Pull request by xmulligan**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -37,7 +37,12 @@ In the absence of conformance testing, it will be at the discretion of Cilium ma
 
 ## Listing Cilium training
 
-We would be delighted to list courses and providers that offer effective [Cilium training](https://cilium.io/enterprise). In the absence of a formal training program (along similar lines to [Kubernetes Training Partners](https://www.cncf.io/certification/training/becoming-a-ktp/)), please reach out on [Cilium Slack](https://cilium.herokuapp.com/) to discuss your offering. 
+In order to be listed as a Cilium Training Partner, the following requirements must be met:
```
Comment (lizrice) [inline CONTRIBUTING.md:40]:
```suggestion
In order to be listed as a [Cilium Training Partner](https://cilium.io/enterprise), the following requirements must be met:
```
Assuming we're still planning to list them there

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -37,7 +37,12 @@ In the absence of conformance testing, it will be at the discretion of Cilium ma
...
+
+* Training partner must have a track record of providing high-quality trainings for open source technology and projects
+* Trainings must cover Cilium Open Source or an approved distribution
```
Comment (lizrice) [inline CONTRIBUTING.md:43]:
```suggestion
* Trainings must cover Cilium Open Source or an [approved distribution](https://cilium.io/enterprise)
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -37,7 +37,12 @@ In the absence of conformance testing, it will be at the discretion of Cilium ma
...
+* The provided Cilium training will be reviewed by a Cilium committer to assess quality, accuracy, and completness.
```
Comment (lizrice) [inline CONTRIBUTING.md:44]:
```suggestion
* The provided Cilium training will be reviewed by a Cilium committer to assess quality, accuracy, and completeness.
```

**Review by lizrice — CHANGES_REQUESTED** [review]
Couple tiny nits but otherwise good to go

**Review by tgraf — APPROVED** [review]

**Review by lizrice — APPROVED** [review]

**Pull request merged**